cmd/eksctl-anywhere/cmd/flags.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+func addClusterFlags(cc *cobra.Command, clusterOpt *clusterOptions, override ...string) {
+	overriddenFlags := make(map[string]struct{})
+	for _, flag := range override {
+		overriddenFlags[flag] = struct{}{}
+	}
+
+	if _, ok := overriddenFlags["filename"]; !ok {
+		cc.Flags().StringVarP(&clusterOpt.fileName, "filename", "f", "", "Filename that contains EKS-A cluster configuration")
+	}
+	if _, ok := overriddenFlags["bundles-override"]; !ok {
+		cc.Flags().StringVar(&clusterOpt.bundlesOverride, "bundles-override", "", "Override default Bundles manifest (not recommended)")
+	}
+	if _, ok := overriddenFlags["kubeconfig"]; !ok {
+		cc.Flags().StringVar(&clusterOpt.managementKubeconfig, "kubeconfig", "", "Management cluster kubeconfig file")
+	}
+	if _, ok := overriddenFlags["install-packages"]; !ok {
+		cc.Flags().StringVar(&clusterOpt.installPackages, "install-packages", "", "Location of curated packages configuration files to install to the cluster")
+	}
+	if _, ok := overriddenFlags["force-cleanup"]; !ok {
+		cc.Flags().BoolVar(&clusterOpt.forceCleanup, "force-cleanup", false, "Force deletion of previously created bootstrap cluster")
+	}
+}
+
+func bindAllFlags(cmd *cobra.Command, args []string) error {
```

Comment:
Can we call this `bindFlagsToViper` given we're using the Viper singleton. I.e. there's nothing in the signature that indicates we're doing anything with Viper as is.

Reply:
Updated the naming to `bindFlagsToViper`